cli/internal/globby/globby.go

```diff
@@ -24,8 +38,10 @@ func GlobFiles(basePath string, includePatterns []string, excludePatterns []stri
 
 	includePattern := "{" + strings.Join(include, ",") + "}"
 	excludePattern := "{" + strings.Join(exclude, ",") + "}"
-	_ = fs.Walk(basePath, func(p string, isDir bool) error {
-		if val, _ := doublestar.PathMatch(excludePattern, p); val {
+
+	_ = afs.Walk(basePath, func(path string, info os.FileInfo, err error) error {
```

Comment:
I suspect we shouldn't ignore this error, even if we were previously doing that. Or if we should, we should explain why it's ok.

Reply:
Since this is only showing up in this diff because of a rename I believe we should defer this to the follow-up PR which is a complete reimplementation of this function and won't, for example, even call `afero.Walk`.

I'll make sure that we don't have dangling unhandled errors in that pass.